impl/client/src/main/javascript/web/prompting/PromptPanel.js

```diff
@@ -1433,15 +1433,19 @@ function(Base, Logger, DojoNumber, i18n, Utils, GUIDHelper, WidgetBuilder, Dashb
         this.submit(this, {isInit: !this.isRefresh});
       }
 
-      if(this.onAfterUpdate) {
-        this.onAfterUpdate();
+      try{
+        if(this.onAfterUpdate) {
+          this.onAfterUpdate();
+        }
+      } catch (error) {
+        console.log(error);
```

Comment:
My goal was simply to mimic previous behavior. In that sense, you should not catch the exception. Just remove the `catch` section.